fabric/src/generated/resources/assets/gtceu/lang/en_us.json

```diff
@@ -2252,6 +2252,60 @@
+  "gtceu.jei.ore_vein_diagram.spawn_range": "Spawn Range:",
+  "gtceu.jei.ore_vein_diagram.weight": "Weight: %s",
+  "gtceu.jei.ore_vein_diagram.dimensions": "Dimensions:",
+  "gtceu.jei.ore_vein_bauxite_vein_end": "Bauxite Vein",
+  "gtceu.jei.ore_vein_magnetite_vein_end": "Magnetite Vein",
+  "gtceu.jei.ore_vein_naquadah_vein": "Naquadah Vein",
+  "gtceu.jei.ore_vein_pitchblende_vein_end": "Pitchblende Vein",
+  "gtceu.jei.ore_vein_scheelite_vein": "Scheelite Vein",
+  "gtceu.jei.ore_vein_sheldonite_vein": "Sheldonite Vein",
+  "gtceu.jei.ore_vein_banded_iron_vein": "Banded Iron Vein",
+  "gtceu.jei.ore_vein_beryllium_vein": "Beryllium Vein",
+  "gtceu.jei.ore_vein_certus_quartz": "Certus Quartz Vein",
+  "gtceu.jei.ore_vein_manganese_vein": "Manganese Vein",
+  "gtceu.jei.ore_vein_molybdenum_vein": "Molybdenum Vein",
+  "gtceu.jei.ore_vein_monazite_vein": "Monazite Vein",
+  "gtceu.jei.ore_vein_nether_quartz_vein": "Nether Quartz Vein",
+  "gtceu.jei.ore_vein_redstone_vein": "Redstone Vein",
+  "gtceu.jei.ore_vein_saltpeter_vein": "Saltpeter Vein",
+  "gtceu.jei.ore_vein_sulfur_vein": "Sulfur Vein",
+  "gtceu.jei.ore_vein_tetrahedrite_vein": "Tetrahedrite Vein",
+  "gtceu.jei.ore_vein_topaz_vein": "Topaz Vein",
+  "gtceu.jei.ore_vein_apatite_vein": "Apatite Vein",
+  "gtceu.jei.ore_vein_cassiterite_vein": "Cassiterite Vein",
+  "gtceu.jei.ore_vein_coal_vein": "Coal Vein",
+  "gtceu.jei.ore_vein_copper_tin_vein": "Copper Tin Vein",
+  "gtceu.jei.ore_vein_galena_vein": "Galena Vein",
+  "gtceu.jei.ore_vein_garnet_tin_vein": "Garnet Tin Vein",
+  "gtceu.jei.ore_vein_garnet_vein": "Garnet Vein",
+  "gtceu.jei.ore_vein_iron_vein": "Iron Vein",
+  "gtceu.jei.ore_vein_lubricant_vein": "Lubricant Vein",
+  "gtceu.jei.ore_vein_magnetite_vein_ow": "Magnetite Vein",
+  "gtceu.jei.ore_vein_mineral_sand_vein": "Mineral Sand Vein",
+  "gtceu.jei.ore_vein_nickel_vein": "Nickel Vein",
+  "gtceu.jei.ore_vein_salts_vein": "Salts Vein",
+  "gtceu.jei.ore_vein_oilsands_vein": "Oilsands Vein",
+  "gtceu.jei.ore_vein_copper_vein": "Copper Vein",
+  "gtceu.jei.ore_vein_diamond_vein": "Diamond Vein",
+  "gtceu.jei.ore_vein_lapis_vein": "Lapis Vein",
+  "gtceu.jei.ore_vein_manganese_vein_ow": "Manganese Vein",
+  "gtceu.jei.ore_vein_mica_vein": "Mica Vein",
+  "gtceu.jei.ore_vein_olivine_vein": "Olivine Vein",
+  "gtceu.jei.ore_vein_redstone_vein_ow": "Redstone Vein",
+  "gtceu.jei.ore_vein_sapphire_vein": "Sapphire Vein",
+  "gtceu.jei.ore_vein_raw_oil_deposit": "Raw Oil Deposit",
+  "gtceu.jei.ore_vein_lava_deposit": "Lava Deposit",
+  "gtceu.jei.ore_vein_salt_water_deposit": "Salt Water Deposit",
+  "gtceu.jei.ore_vein_heavy_oil_deposit": "Heavy Oil Deposit",
+  "gtceu.jei.ore_vein_oil_deposit": "Oil Deposit",
+  "gtceu.jei.ore_vein_natural_gas_deposit": "Natural Gas Deposit",
+  "gtceu.jei.ore_vein_nether_natural_gas_deposit": "Natural Gas Deposit",
+  "gtceu.jei.ore_vein_light_oil_deposit": "Light Oil Deposit",
```

Comment:
did you just edit the lang file for these? won't work. register them in a loop in `IntegrationLang`, and also, do the key in `gtceu.jei.ore_vein.<veinname>` instead of `gtceu.jei.ore_vein_<veinname>`